### ISSUE TITLE
google colab shader error

### ISSUE THREAD

**Issue by shahik** [podgorskiy/ALAE#13]
Hi, 
    Congratulations & thanks for this paper! I am running interactive_demo on colab and getting non-stop error

`
ERROR: ImGui_ImplOpenGL3_CreateDeviceObjects: failed to compile fragment shader!
ERROR: ImGui_ImplOpenGL3_CreateDeviceObjects: failed to link shader program!
ERROR: ImGui_ImplOpenGL3_CreateDeviceObjects: failed to compile vertex shader!
ERROR: ImGui_ImplOpenGL3_CreateDeviceObjects: failed to compile fragment shader!
ERROR: ImGui_ImplOpenGL3_CreateDeviceObjects: failed to link shader program!
ERROR: ImGui_ImplOpenGL3_CreateDeviceObjects: failed to compile vertex shader!
ERROR: ImGui_ImplOpenGL3_CreateDeviceObjects: failed to compile fragment shader!
`
Any solution? 
Greetings!

**Comment by TheNarrator** [comment]
I also got the same error connecting through SSH. The error didn't happen when run locally. The **bimpy** package probably needs to connect to a display to work.

**Comment by podgorskiy** [comment]
Yes, bimpy needs system X to be installed. It won't work on headless machines.
You still can create a colab demo, by removing all dependencies on bimpy and display images statically, but then it won't be interactive.

I'll work on making a remote version of bimpy that would allow interacting with it through browser in the nearest future.

**Comment by AlexRMU** [comment]
@podgorskiy 
Write here here when you're done, please

**Comment by shahik** [comment]
@podgorskiy  Plz do make it 🙏
